python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -1167,6 +1175,43 @@ def matchCatalog(self, catalog, filterName, astrometryConfig):
         matches = matchNanojanskyToAB(matches)
         return joinMatches(matches, "ref_", "src_")
 
+    def plotRhoStatistics(self, catalog, filenamer, dataId, butler=None, camera=None, ccdList=None,
+                          tractInfo=None, patchList=None, hscRun=None, matchRadius=None, zpLabel=None,
+                          forcedStr=None, postFix="", flagsCat=None, uberCalLabel=None):
+        enforcer = None
+        unitStr = " (arcsec) "
+        plotAllKwargs = dict(butler=butler, camera=camera, ccdList=ccdList, tractInfo=tractInfo,
+                             patchList=patchList, hscRun=hscRun, matchRadius=matchRadius,
+                             zpLabel=zpLabel, forcedStr=forcedStr, uberCalLabel=uberCalLabel)
+
+        treecorr_kwargs = {'nbins':11, 'min_sep':0.5, 'max_sep':20, 'sep_units':"arcmin", 'verbose':0}
```

Comment:
these are also hardcoded there: https://github.com/lsst-dm/pipe_analysis/blob/tickets/DM-23539/python/lsst/pipe/analysis/utils.py#L432

This could be dangerous if either was to change. Could these be configurable, with these values being the default? If not, they should still definitely only be set once. (see also comment about `max_sep` in `plotRhoStatistics.py`

Reply:
I am not sure about 'dangerous' comment, as there is a clear hierarchy: the defaults in utils.py kick in only if they were not set visitAnalysis or coaddAnalysis. I will make their values configurable in the visitAnalysis and coaddAnalysis scripts, but leave them in utils.py hardcoded? Or should I remove the default values from there altogether?

Reply:
I think for those we want to be configurable, it makes more sense to have the value always read from config, with _that_ providing the default you want?